### PR TITLE
Automated ticker

### PR DESCRIPTION
**Pull request by tomrf1**
## Why are you doing this?
Use the new automated ticker value

[**Trello Card**](https://trello.com)

## Changes

* Change 1
* Change 2

### CODE REVIEW

File: conf/CODE.public.conf [hunk 1]
```diff
@@ -16,7 +16,7 @@ support.url="https://support.code.dev-theguardian.com"
...
+play.filters.headers.contentSecurityPolicy = "default-src 'self' www.paypalobjects.com www.paypal.com www.sandbox.paypal.com js.stripe.com pasteup.guim.co.uk ophan.theguardian.com j.ophan.co.uk media.guim.co.uk www.google-analytics.com www.googletagmanager.com tagmanager.google.com polyfill.io www.googleadservices.com googleads.g.doubleclick.net www.google.com www.google.co.uk optimize.google.com static.ads-twitter.com bat.bing.com bid.g.doubleclick.net t.co analytics.twitter.com stats.g.doubleclick.net www.youtube-nocookie.com connect.facebook.net www.facebook.com consumer.krxd.net cdn.krxd.net beacon.krxd.net secure.adnxs.com checkout.stripe.com fonts.googleapis.com ssl.gstatic.com www.gstatic.com fonts.gstatic.com sentry.io www.dwin1.com data: wss: 'unsafe-inline' q.stripe.com payment.code.dev-guardianapis.com https://interactive.guim.co.uk/ https://support.theguardian.com/"
```
Comment (joelochlann) [inline conf/CODE.public.conf]: weird that we need it to whitelist itself! @Ap0c knows a bit more than me about this Content Security Policy - is this expected behaviour?

File: assets/pages/new-contributions-landing/components/ContributionUsTicker.jsx [hunk 2]
```diff
@@ -16,11 +16,11 @@ type PropTypes = {}
 // ---- Helpers ----- //
 
 const getInitialTickerValues = (): Promise<StateTypes> =>
-  fetch('https://interactive.guim.co.uk/docsdata-test/1ySn7Ol2NQLvvSw_eAnVrPuuRnaGOxUmaUs6svtu_irU.json')
+  fetch('https://support.theguardian.com/ticker.json')
```
Comment (joelochlann) [inline assets/pages/new-contributions-landing/components/ContributionUsTicker.jsx]: can you not just do `/ticker.json`? then it will work in CODE & dev

Comment (joelochlann) [reply assets/pages/new-contributions-landing/components/ContributionUsTicker.jsx]: and maybe it won't need the addition to the `contentSecurityPolicy`

Comment (tomrf1) [reply assets/pages/new-contributions-landing/components/ContributionUsTicker.jsx]: very good point!

**Pull request merged**